modules/supervisor/src/reachingSupervisor.cpp

```diff
@@ -144,6 +146,7 @@ bool reachingSupervisor::updateModule()
 
         printf("===============================\n");
         printf("[%s] updateModule()\n", name.c_str());
+        gotPlan = true;
```

Comment:
This triggers the flag when there is new `motionPlan` message received in the `planPortIn`